src/openafval/trash/admin.py

```diff
@@ -0,0 +1,118 @@
+from django.contrib import admin
+from django.http import HttpRequest
+from django.utils.translation import gettext_lazy as _
+
+from .models import (
+    BagObject,
+    Container,
+    ContainerType,
+    Emptying,
+    Entity,
+    EntityObjectManagement,
+)
+
+
+class ReadOnlyMixin:
+    self: admin.ModelAdmin
+
+    def has_change_permission(self, request: HttpRequest, obj=None):
+        return False
+
+    def has_add_permission(self, request: HttpRequest, obj=None):
+        return False
+
+    def has_delete_permission(self, request: HttpRequest, obj=None):
+        return False
+
+
+@admin.register(BagObject)
+class BagObjectAdmin(ReadOnlyMixin, admin.ModelAdmin):
+    list_display = (
+        "identifier",
+        "address",
+    )
+
+
+@admin.register(Entity)
+class EntityAdmin(ReadOnlyMixin, admin.ModelAdmin):
```

Comment:
- As with the model, "Entity" is not a good term for anything, since it applies to everything without distinction.
- Consider adding `search_fields = ("bsn", "name", "address", "postcode")` and `list_filter = ("postcode",)`